requirements.txt

```diff
@@ -17,7 +18,7 @@ Flask==1.1.1
 Flask-Cors==3.0.8
 gevent==1.4.0
 greenlet==0.4.15
-gunicorn[gevent,setproctitle]==20.0.4
+gunicorn==20.0.4
```

Comment:
looks wrong

Reply:
The real requirements are in setup.cfg, requirements.txt can be generated by `tox -egenreq` and it drops extras just like dependabot does.